tensorizer/serialization.py

```diff
@@ -3510,7 +3510,7 @@ def _mv_suffix(data: "collections.abc.Buffer", start: int):
         if not isinstance(data, memoryview):
             data = memoryview(data)
         try:
-            if data.ndim != 1:
+            if data.ndim != 1 or data.format != "B":
```

Comment:
Would casting a byte format to a byte format be a no-op, effectively?

Reply:
It would create an intermediate `memoryview` object that is referenced only by the one returned by this function. So it's not quite a no-op, but the output would be able to be used in the same way.